php-openvpn-connection-manager/debian/changelog

```diff
@@ -1,6 +1,12 @@
+php-openvpn-connection-manager (1.0.2-6) stable; urgency=medium
+
+  * debian/control: enhance description.
+
+ -- Joost van Baal-Ilić <joostvb-eduvpn@ad1810.com>  Sat, 17 Nov 2018 10:33:43 +0100
+
 php-openvpn-connection-manager (1.0.2-5) stable; urgency=medium
 
-  * debian/control: make pkg-php-tools Build-Depends versioned, since we use
+  * debian/control: make pkg-p-tools Build-Depends versioned, since we use
```

Comment:
pkg-p-tools ?

Reply:
oops, typo indeed. eyeballed diff again, the one you've spotted is the only one. could you please fix & merge?